README.md

```diff
@@ -1,7 +1,172 @@
-# VisualMQ
 
-## Development
-1. Clone this repo.
-1. Open the folder in Unity as a new project.
-1. Create and develop :art:
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![License][license-shield]][license-url]
 
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/VisualMQ/VisualMQ/">
+    <img src="visualmq-logo.png" alt="Logo">
+  </a>
+
+  <h3 align="center">VisualMQ</h3>
```

Comment:
let's use `IBM VisualMQ` to make it consistent

Reply:
https://www.ibm.com/legal/copytrade

Have we confirmed to have permission from IBM to use this?